README.md

```
@@ -1,36 +1,97 @@
-### 💬 6. Feedback and Iteration
-Reviewers may leave comments or suggestions. Update your branch and push changes until the PR is approved.
+---
+
+## Authorization & Access Control
+
+- All endpoints require authentication.
+- Unauthenticated users will only see the login page or an error response.
+- **ADMIN** users can access all dogs and all training sessions.
+- **USER** users can only access dogs and trainings that they own.
+- Endpoints requiring elevated roles (USER or ADMIN):
+    - `POST /api/dogs`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)